packages/elastic_agent/manifest.yml

```
@@ -1,14 +1,14 @@
 name: elastic_agent
 title: Elastic Agent
-version: 1.19.0
+version: 1.20.0
 description: Collect logs and metrics from Elastic Agents.
 type: integration
 format_version: 1.0.0
```

I would suggest updating to a recent spec versions, there are many validations that we are missing by using an old version.
```suggestion
format_version: 3.1.4
```
But feel free to do this in a separate PR as it will likely require more changes.

---

That triggered bunch of errors, I'll fix them on their own PR and once it's merged, I'll rebase this onto `main`.

---

I'm getting many errors like those:
```
   1. file "/home/tiago/devel/integrations/packages/elastic_agent/data_stream/apm_server_metrics/fields/fields.yml" is invalid: field 1.fields.3.fields.3.fields.5: object_type is required
   2. file "/home/tiago/devel/integrations/packages/elastic_agent/data_stream/auditbeat_metrics/fields/fields.yml" is invalid: field 1.fields.3.fields.3.fields.5: object_type is required
   3. file "/home/tiago/devel/integrations/packages/elastic_agent/data_stream/cloud_defend_logs/fields/fields.yml" is invalid: field 1: object_type is required
   4. file "/home/tiago/devel/integrations/packages/elastic_agent/data_stream/cloud_defend_logs/fields/fields.yml" is invalid: field 2: object_type is required
```

@jsoriano do you have any idea what is missing? The [update doc](https://github.com/elastic/elastic-package/blob/main/docs/howto/update_major_package_spec.md) says `object_type` can be omitted, so I'm quite confused here.